packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/FormDescriptionEditorRepresentation.tsx

```diff
@@ -141,11 +132,19 @@ const useFormDescriptionEditorStyles = makeStyles((theme) => ({
   },
 }));
 
+export type WidgetDescriptor = {
```

Comment:
WidgetDescriptor seems declared here, and also in FormDescriptionEditorRepresentation.types.ts

Reply:
done.